test/base/SpecBase.scala

```diff
@@ -57,7 +60,9 @@ trait SpecBase extends PlaySpec with GuiceOneAppPerSuite with TryValues with Moc
       .overrides(
         bind[DataRequiredAction].to[DataRequiredActionImpl],
         bind[IdentifierAction].to(fakeIdentifierAction),
-        bind[DataRetrievalAction].toInstance(new FakeDataRetrievalAction(userAnswers))
+        bind[UTRAuthenticationAction].to[FakeUTRAuthenticationAction],
+        bind[DataRetrievalAction].toInstance(new FakeDataRetrievalAction(userAnswers)),
+        bind[CountryOptions].toInstance(nonUkCountryOptions)
```

Comment:
It seems weird to me to be binding CountryOptions anyway, since it seems like either using UK or non-UK country options would be a per-case choice, not a global setting. However, even beyond that, it seems weird to be to binding the non-UK country options in the test when the normal module binds the UK ones. Why would it be different in the test?